config.go

```diff
@@ -15,90 +14,24 @@ import (
 const configName = ".phraseapp.yml"
 const defaultDir = "./"
 
-type Credentials struct {
-	phraseapp.Credentials
-	Username string `cli:"opt --username -u desc='username used for authentication'"`
-	Token    string `cli:"opt --access-token -t desc='access token used for authentication'"`
-	TFA      bool   `cli:"opt --tfa desc='use Two-Factor Authentication'"`
-	Host     string `cli:"opt --host desc='Host to send Request to'"`
-	Debug    bool   `cli:"opt --verbose -v desc='Verbose output'"`
-}
-
-func ConfigCallArgs() (map[string]string, error) {
+func ReadConfig() (*phraseapp.Config, error) {
 	content, err := ConfigContent()
 	if err != nil {
-		content = "{}"
+		content = []byte("{}")
 	}
 
-	return parseCallArgs(content)
+	cfg := new(phraseapp.Config)
+	return cfg, yaml.Unmarshal(content, cfg)
 }
 
-func parseCallArgs(yml string) (map[string]string, error) {
-	var callArgs *CallArgs
-
-	err := yaml.Unmarshal([]byte(yml), &callArgs)
-	if err != nil {
-		return nil, err
-	}
-
-	m := make(map[string]string)
-
-	if callArgs != nil {
-		m["ProjectID"] = callArgs.Phraseapp.ProjectID
-		m["AccessToken"] = callArgs.Phraseapp.AccessToken
-	}
-
-	return m, nil
-}
-
-func ClientFromCmdCredentials(c Credentials) (*phraseapp.Client, error) {
-	defaultCredentials, e := ConfigDefaultCredentials()
-	if e != nil {
-		return nil, e
-	}
-
-	return phraseapp.NewClient(PhraseAppCredentials(c), defaultCredentials)
-}
-
-func PhraseAppCredentials(c Credentials) phraseapp.Credentials {
-	return phraseapp.Credentials{
-		Username: c.Username,
-		Token:    c.Token,
-		TFA:      c.TFA,
-		Host:     c.Host,
-		Debug:    c.Debug,
-	}
-}
-
-func ConfigDefaultCredentials() (*phraseapp.Credentials, error) {
-	content, err := ConfigContent()
-	if err != nil {
-		content = "{}"
-	}
-
-	return parseCredentials(content)
-}
-
-func ConfigDefaultParams() (phraseapp.DefaultParams, error) {
-	content, err := ConfigContent()
-	if err != nil {
-		content = "{}"
-	}
-
-	return parseDefaults(content)
-}
 
-func ConfigContent() (string, error) {
+func ConfigContent() ([]byte, error) {
 	path, err := configPath()
 	if err != nil {
-		return "", err
+		return []byte(""), err
```

Comment:
why not just return `nil, err`. It seems parsing `""` leads to something we might nit want:

```
$ ruby -r yaml -e 'p YAML.parse("").inspect'
"false"
```